_bibliography/references.bib

```diff
@@ -1,4 +1,21 @@
 
+@article{bachmann_enrichment_2021,
+	title = {Enrichment dynamics for advanced reactor {HALEU} support},
+	volume = {7},
+	copyright = {© A. M. Bachmann et al., Published by EDP Sciences, 2021},
+	issn = {2491-9292},
+	url = {https://www.epj-n.org/articles/epjn/abs/2021/01/epjn210024/epjn210024.html},
+	doi = {10.1051/epjn/2021021},
+	abstract = {Transitioning to High Assay Low Enriched Uranium-fueled reactors will alter the material requirements of the current nuclear fuel cycle, in terms of the mass of enriched uranium and Separative Work Unit capacity. This work simulates multiple fuel cycle scenarios using Cyclus to compare how the type of the advanced reactor deployed and the energy growth demand affect the material requirements of the transition to High Assay Low Enriched Uranium-fueled reactors. Fuel cycle scenarios considered include the current fleet of Light Water Reactors in the U.S. as well as a no-growth and a 1\% growth transition to either the Ultra Safe Nuclear Corporation Micro Modular Reactor or the X-energy Xe-100 reactor from the current fleet of U.S. Light Water Reactors. This work explored parameters of interest including the number of advanced reactors deployed, the mass of enriched uranium sent to the reactors, and the Separative Work Unit capacity required to enrich natural uranium for the reactors. Deploying Micro Modular Reactors requires a higher average mass and Separative Work Unit capacity than deploying Xe-100 reactors, and a lower enriched uranium mass and a higher Separative Work Unity capacity than required to fuel Light Water Reactors before the transition. Fueling Xe-100 reactors requires less enriched uranium and Separative Work Unit capacity than fueling Light Water Reactors before the transition.},
+	language = {en},
+	urldate = {2021-12-02},
+	journal = {EPJ Nuclear Sciences \& Technologies},
+	author = {Bachmann, Amanda M. and Fairhurst-Agosta, Roberto and Richter, Zoë and Ryan, Nathan and Munk, Madicken},
+	year = {2021},
+	note = {Publisher: EDP Sciences},
+	pages = {22},
+	file = {Full Text PDF:/home/amandabachmann/Zotero/storage/FGPXXI8Q/Bachmann et al. - 2021 - Enrichment dynamics for advanced reactor HALEU sup.pdf:application/pdf;Snapshot:/home/amandabachmann/Zotero/storage/G7H5GX2U/epjn210024.html:text/html},
```

Comment:
I think because this is a unique directory structure to your computer we should  make a practice of removing it.